WebRoot/public/hostedlogin.jsp

```diff
@@ -277,7 +312,7 @@ if (application.getInitParameter("offlineMode") != null)  {
 													<option value="standard"  <c:if test="${client eq 'standard'}">selected</c:if>> <fmt:message key="clientStandard"/></option>
                                                     <option value="mobile"  <c:if test="${client eq 'mobile'}">selected</c:if>> <fmt:message key="clientMobile"/></option>
 												</select>
-												
```

Comment:
Some white spaces are removed in pkg-builder.pl, hostedlogin.jsp and login.jsp.
Please fix only conflicts, and don't change any other parts while rebasing.
We may face conflicts when we rebase snickers again (in the future).

Reply:
it might be because of editor 
I didn't remove any thing except conflicts, about white spaces, less it can be possible that due to editor setting. If it a big deal, please let me know I will check again and fix them too.

Reply:
Please configure editor not to remove a white space/tab at the end of line.